package.json

```diff
@@ -26,7 +26,7 @@
     "@types/dompurify": "^2.3.3",
     "@types/jsdom": "^16.2.14",
     "@types/node": "^16.11.27",
-    "@utrecht/component-library-css": "1.0.0-alpha.222",
+    "@utrecht/component-library-css": "^1.0.0-alpha.361",
```

Comment:
versie 1 is alpha? hier is het dan wel verstandig om ^ weg te halen (ook bij de volgende regel)

Reply:
Goeie, done